dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/startup/SchedulerStart.java

```diff
@@ -48,29 +48,44 @@
 
 /**
  *
- * Reschedule old jobs and execute jobs which were scheduled when the server was not running.
+ * Reschedule old jobs and execute jobs which were scheduled when the server was
+ * not running.
  *
  * @author Henning Håkonsen
  */
-public class SchedulerStart
-    extends AbstractStartupRoutine
+public class SchedulerStart extends AbstractStartupRoutine
 {
     private static final Log log = LogFactory.getLog( SchedulerStart.class );
 
     private final String CRON_HOURLY = "0 0 * ? * *";
+
     private final String CRON_DAILY_2AM = "0 0 2 ? * *";
+
     private final String CRON_DAILY_7AM = "0 0 7 ? * *";
 
+    private final String LEADER_JOB_CRON_FORMAT = "0 0/%s * * * *";
+
     private final String DEFAULT_FILE_RESOURCE_CLEANUP = "File resource clean up";
+
     private final String DEFAULT_DATA_STATISTICS = "Data statistics";
+
     private final String DEFAULT_VALIDATION_RESULTS_NOTIFICATION = "Validation result notification";
+
     private final String DEFAULT_CREDENTIALS_EXPIRY_ALERT = "Credentials expiry alert";
+
     private final String DEFAULT_DATA_SET_NOTIFICATION = "Dataset notification";
+
     private final String DEFAULT_REMOVE_EXPIRED_RESERVED_VALUES = "Remove expired reserved values";
 
+    private final String DEFAULT_LEADER_ELECTION = "Leader election in cluster";
+
     @Autowired
     private SystemSettingManager systemSettingManager;
 
+    private String redisEnabled;
```

Comment:
Does these have to be methods and set by the bean?

Reply:
I mean, is it necessary to set the fields? Could they be "static" like: 
private static String redisEnabled="redisEnabled"